src/ts/layouts/LoginLayout.tsx

```diff
@@ -1,27 +1,24 @@
-import React, { Component } from 'react'
-import { connect } from 'react-redux'
-import Image from 'semantic-ui-react/dist/commonjs/elements/Image/Image'
-import { IAppState } from '../background/store/all'
-import { LayoutContainer } from '../components/basic-components'
+import React, {Component} from 'react'
```

Comment:
there are a lot format changes here which shouldn't be applied. Do you use the tslint in our repo in your IDE?

Reply:
I do run tslint, but it must be from 'reformat code' from Webstorm IDE.